test/support/api/patients.js

```diff
@@ -31,3 +31,33 @@ Cypress.Commands.add('routePatient', (mutator = _.identity) => {
   })
     .as('routePatient');
 });
+
+Cypress.Commands.add('routePatients', (mutator = _.identity) => {
+  cy
+    .fixture('collections/patients').as('fxPatients')
+    .fixture('collections/groups').as('fxGroups');
+
+  cy.route({
+    url: '/api/patients**', // probably a regex to match for includes or just copy/past it?
```

Comment:
@paulfalgout what should this be?

Reply:
probably `/api/patients?*` otherwise you're going to match single patients too